src/Dictionaries/ExecutableDictionarySource.h

```diff
@@ -15,10 +15,19 @@ namespace DB
 class ExecutableDictionarySource final : public IDictionarySource
 {
 public:
+
+    struct Configuration
+    {
+        bool implicit_key;
```

Comment:
Maybe add comments here.

Reply:
Will add in separate pull request, need also update docs.